onnxruntime/wasm/api.h

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <emscripten.h>
+
+#include <stddef.h>
+
+namespace Ort {
+struct Session;
+}
+typedef Ort::Session* ort_session_handle_t;
+
+struct OrtValue;
+typedef OrtValue* ort_tensor_handle_t;
+
+extern "C" {
+
+/**
+ * perform global initialization. should be called only once.
+ */
+void EMSCRIPTEN_KEEPALIVE ort_init();
+
+/**
+ * create an instance of ORT session.
+ * @param data a pointer to a buffer that contains the model.
+ * @param data_length the size of the buffer in bytes.
+ * @returns a handle of the ORT session.
+ */
+ort_session_handle_t EMSCRIPTEN_KEEPALIVE ort_create_session(void* data, size_t data_length);
+
+/**
+ * release the specified ORT session.
+ */
+void EMSCRIPTEN_KEEPALIVE ort_release_session(ort_session_handle_t session);
+
+size_t EMSCRIPTEN_KEEPALIVE ort_get_input_count(ort_session_handle_t session);
+size_t EMSCRIPTEN_KEEPALIVE ort_get_output_count(ort_session_handle_t session);
+
+/**
+ * get the model's input name.
+ * @param session handle of the specified session
+ * @param index the input index
+ * @returns a pointer to a buffer which contains C-style string.
+ * @remarks caller should release the C style string after use by calling ort_free().
+ */
```

Comment:
Is it 'should' or 'must'? If 'must' maybe add this to the 'returns' text as it's an important detail.